packages/shared/src/components/modals/common/Modal.tsx

```diff
@@ -70,7 +70,7 @@ export const modalSizeToClassName: Record<ModalSize, string> = {
   [ModalSize.XSmall]: 'w-[21.25rem]',
   [ModalSize.Small]: 'w-[26.25rem]',
   [ModalSize.Medium]: 'w-[35rem]',
-  [ModalSize.Large]: 'w-[42.5rem]',
+  [ModalSize.Large]: '!w-[42.5rem]',
```

Comment:
Unfortunately we use this for the actual page, which also has a w-full so always get's overruled.

Reply:
Oh. So the tailwind update prioritizes pre-defined values or arbitrary values now?